src/Common.am

```diff
@@ -73,3 +74,21 @@ COMPAT_LIB = $(top_builddir)/compat/libcompatsquid.la $(LIBPROFILER)
 
 ## Some helpers are written in Perl and need the local shell defined properly
 subst_perlshell = sed -e 's,[@]PERL[@],$(PERL),g' <$(srcdir)/$@.pl.in >$@ || ($(RM) -f $@ ; exit 1)
+
+EXTENSIONS=.hdrtest
+.h.hdrtest:
+	@srcfilepath=`echo $< | $(SED) 's%^$(top_srcdir)/%%'`; \
+	srcfile=`basename $$srcfilepath`; \
+	tgtfile=`basename $@`; \
+	if test "" != "$(DONOTTEST)" && echo "$(DONOTTEST)" | $(TR) ' ' '\n' | $(GREP) -x -q "$$srcfile" ; then \
```

Comment:
Perhaps making sure that all DONOTTEST entries are surrounded by spaces can be used to simplify this?

```suggestion
	if test "$(DONOTTEST)" != "" && echo " $(DONOTTEST) " | $(GREP) -F -q " $$srcfile " ; then \
```

The above suggestion also avoids Yoda speak in the first condition (and makes it more consistent with the existing code in our Makefile.am files).

Reply:
Your call. IMHO, many readers will not even know what `tr` or `grep -x` do (not to mention what they "signal") :-).

Adding `-F` (if it is considered as portable as `-x`) is a good idea regardless AFAICT because srcfile should not be interpreted as a regular expression.

Reply:
yes. That was a leftover of experimentation. Using $(FGREP) was the original intent